site/en/blog/deprecating-web-sql/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Deprecating Web SQL
```

Comment:
And here I've always called it "WebSQL", without the space. Looks like people on the Internet do it both ways.